docs/admin/access.rst

```diff
@@ -187,59 +189,60 @@ to a containing project as well.
 An additional step is performed if checking permission for the translation:
 
 
-4. Group :guilabel:`Languages` are matched against accessed translations, it is ignored for component or project level access.
+4. Group :guilabel:`Languages` are matched against accessed translations, it is ignored for component- or project-level access.
 
 .. hint::
 
-   You can use :guilabel:`Language selection` or :guilabel:`Project selection`
+   Use :guilabel:`Language selection` or :guilabel:`Project selection`
    to automate inclusion of all languages or projects.
 
 Checking access to a project
 ++++++++++++++++++++++++++++
 
 A user has to be a member of a group linked to the project or any component
-inside it. Only membership is enough, no specific permissions are needed to
+inside it. Only having membership is enough, no specific permissions are needed to
 access a project (this is used in the default `Viewers` group, see
 :ref:`default-groups`).
 
 Checking access to a component
 ++++++++++++++++++++++++++++++
 
-A user can access the unrestricted component once he can access the containing
-project. With enabled :ref:`component-restricted` the access to the component
-requires explicit permission to the component (or containing component list).
+A user can access the unrestricted component once able to access the containing
+project. With :ref:`component-restricted` turned on, access to the component
+requires explicit permission to it (or a component list it is in).
 
 .. _manage-users:
 
 Managing users and groups
 -------------------------
 
-All users and groups can be managed using the Django admin interface,
-available under :file:`/admin/` URL.
+All users and the various groups they are in can be managed using the
+Django admin interface available, which you can get to by appending
+:file:`/admin/` to the Weblate site URL.
 
 .. _manage-acl:
 
-Managing per project access control
+Managing per-project access control
 +++++++++++++++++++++++++++++++++++
 
 .. note::
 
-    This feature only works for ACL controlled projects, see :ref:`acl`.
+    This feature only works for projects using access-control lists, see :ref:`acl`.
```

Comment:
Why is ":ref:`acl`." not "access-control lists"?

Reply:
@nijel please check

Reply:
ok by me once this is resolved

Reply:
I'm not certain that I get the question. The chapter is called "Project access control" and that's what will be rendered as link to it here. If you change the chapter title, the link will change as well.

Reply:
well, @comradekingu you know what to do if you want to rename this.